lrauv_ignition_plugins/test/test_mission_depth_vbs.cc

```diff
@@ -53,23 +53,22 @@ TEST_F(LrauvTestFixture, DepthVBS)
         lrauvRunning);
   });
 
+  int maxIterations{28000};
```

Comment:
I realize this line is not new, but maybe a comment saying why specifically 28000, even if it's just empirically the number that works for all the missions? Same for other files.

Reply:
Documented on ad14dbf